pkg/scanner/scanner_test.go

```diff
@@ -424,6 +439,60 @@ func TestCheckPVRelevance(t *testing.T) {
 			wantRelevant: true,
 			wantBucket:   testBucketName,
 		},
+		{
+			name:           "Relevant - Override mode",
```

Comment:
Nit: Include the Scenario and Expected Outcome in Test Names https://engdoc.corp.google.com/eng/doc/tott/episodes/660.md?cl=head

by looking at test name, you should be able to see scenario and  expected outcome. You only include scenario in these unit test names.